ui/src/app/subnets/views/SpaceDetails/SpaceSummary/SpaceSummaryForm/SpaceSummaryForm.tsx

```diff
@@ -0,0 +1,61 @@
+import { useDispatch, useSelector } from "react-redux";
+import * as Yup from "yup";
+
+import FormikField from "app/base/components/FormikField";
+import FormikForm from "app/base/components/FormikForm";
+import { actions as spaceActions } from "app/store/space";
+import spaceSelectors from "app/store/space/selectors";
+import type { Space } from "app/store/space/types";
+
+export type SpaceSummaryValues = Pick<Space, "name" | "description">;
+const spaceSummaryFormSchema = Yup.object().shape({
+  name: Yup.string(),
+  description: Yup.string(),
+});
+
+const SpaceSummaryForm = ({
+  space,
+  handleDismiss,
+}: {
+  space: Space;
```

Comment:
We generally prefer to pass the id around only. Our philosophy is if it can be selected from the store that is better than passing lots of data around as props. Amongst the benefits is that this prevents a lot of cascading rerenders.

Reply:
Whilst I'd generally agree, this component seems to simple to justify the additional complexity.

It would mean I'd need to duplicate a lot of logic that the SpaceSummary component already handles (fetching spaces, states for loading not found etc.) and do the same in SpaceDetailsHeader for consistency.

I'll merge this as is for now, but am I'm open to discuss more and make more changes afterwards if you feel strongly about it.

Reply:
I think it would be fine not to replicate those additional states in this case as they would be handling states that couldn't happen in reality.